projects/hello_adc/src/hello_adc.c

```diff
@@ -0,0 +1,39 @@
+#include "adc.h"
+#include "delay.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+
+#define BUTTON_PIN 178
+#define READ_FROM_PIN 166
+
+GpioItCallback prv_log_pin_data(GpioAddress *button_addr, void *context);
+
+int main() {
```

Comment:
This parameter list should be `(void)` rather than `()` - in C, [weird things](https://stackoverflow.com/questions/13950642/why-does-a-function-with-no-parameters-compared-to-the-actual-function-definiti) can happen with functions taking empty parameter lists like in other languages rather than explicitly declaring them as `(void)`.